lib/active_merchant/billing/gateways/blue_snap.rb

```diff
@@ -95,7 +95,7 @@ def capture(money, authorization, options={})
         commit(:capture, :put) do |doc|
           add_authorization(doc, authorization)
           add_order(doc, options)
-          add_amount(doc, money, options)
+          add_amount(doc, money, options) if options[:partial_amount] == true
```

Comment:
@curiousepic I believe I addressed what you were talking about, let me know if I missed what you meant.

Reply:
Wow, I'm sorry, in fact I just misunderstood the original, it was fine. Though I think the option name could be more explicit, since "amount" kinda implies it should be a the actual amount, rather than a boolean flag. Perhaps "partial_capture"  or "capture_partial_amount", or even "include_capture_amount" (since it doesn't necessarily need to be partial)...

Reply:
Confusion is never a good sign. I think I like `include_capture_amount` since it is the most descriptive of what is happening here. I have re-named it so.